src/vs/editor/browser/widget/diffReview.ts

```diff
@@ -780,7 +780,8 @@ export class DiffReview extends Disposable {
 			config.viewInfo.stopRenderingLineAfter,
 			config.viewInfo.renderWhitespace,
 			config.viewInfo.renderControlCharacters,
-			config.viewInfo.fontLigatures
+			config.viewInfo.fontLigatures,
+			undefined
```

Comment:
I couldn’t figure out when this code path was triggered, is text selectable in this case?

Reply:
This is triggerable via `F7` in a diff editor. Not rendering selection there, so that should be fine. You can use a `null` safely...